### PR TITLE
Datastore

### PR DESCRIPTION
**Pull request by ayamkito**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/** Servlet responsible for creating new tasks. */
```
Comment (darshanmodani-sps) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: This comment needs to be changed to what you are doing in the function :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h1>KIZITO AKWOTIGA</h1>
                     <p>COMPUTER ENGINEER</p>
                     <a href="#" class="side-menu">ABOUT ME</a>
                     <a href="#" class="side-menu">RESUME</a>
-                    <a href="#" class="side-menu">GET IN TOUCH</a>
+                    <a href="./contactMe.html" class="side-menu">GET IN TOUCH</a>
```
Comment (darshanmodani-sps) [inline portfolio/src/main/webapp/index.html]: Is this correctly working ?
I believe the href value should just be "/contactMe.html"

**Review by darshanmodani-sps — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 1 above)
Comment (ayamkito) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: Thank you for pointing that out. I will fix it.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (ayamkito) [reply portfolio/src/main/webapp/index.html]: Surprisingly it works but will check your suggestion as well.

**Review by darshanmodani-sps — APPROVED** [review]

**Pull request merged**